### PR TITLE
rgo2019 allow sdf models

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: src/plugin.cpp [hunk 1]
```diff
@@ -132,7 +132,7 @@ void RobocupPlugin::initialize(ed::InitData& init)
             ed::UpdateRequest req;
             std::stringstream error;
 
-            if (!model_loader_.create(id, type, req, error))
+            if (!model_loader_.create(id, type, req, error, true))
```
Comment (LoyVanBeek) [inline src/plugin.cpp]: What does `true` mean here?

Comment (MatthijsBurgh) [reply src/plugin.cpp]: that it also looks for SDF models. Not only yamls

Comment (LoyVanBeek) [reply src/plugin.cpp]: Plz put that in a comment or if possible like a keyword-arg in python: `use_sdf=true`

Comment (MatthijsBurgh) [reply src/plugin.cpp]: It is documented in the library. No need to put it here.

**Pull request merged**